guides/common/assembly_provisioning-cloud-instances-azure.adoc

```diff
@@ -7,7 +7,7 @@ ifdef::context[:parent-context: {context}]
 :context: azure-provisioning
 :CRname: Microsoft Azure Resource Manager
 
-{ProjectName} can interact with Microsoft Azure Resource Manager, including creating new virtual machines and controlling their power management states. Only image-based provisioning is supported for creating Azure hosts.
+{ProjectName} can interact with Microsoft Azure Resource Manager, including creating new virtual machines and controlling their power management states. Only image-based provisioning, including provisioning using custom images is supported for creating Azure hosts.
```

Comment:
Can we make it:
```suggestion
{ProjectName} can interact with Microsoft Azure Resource Manager, including creating new virtual machines and controlling their power management states. Only image-based provisioning, including provisioning using marketplace images, custom images and shared image gallery is supported for creating Azure hosts.
```

Reply:
Thanks for reviewing, Aditi. I've broken it into two sentences for better readability and pushed the change. Hope it makes sense: "Only image-based provisioning is supported for creating Azure hosts. This includes provisioning using marketplace images, custom images, and shared image gallery."